scripts/diagnostic-settings-activity-log-at-subscriptions.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#Script to create or remove DataGuard's Azure Diagnostic Settings from selected Subscriptions.
+
+SAVEIFS=$IFS
+IFS=";"
+LOGS='[{"category": "Administrative","enabled": true},{"category": "Security","enabled": true},{"category": "ServiceHealth","enabled": true},{"category": "Alert","enabled": true},{"category": "Recommendation","enabled": true},{"category": "Policy","enabled": true},{"category": "Autoscale","enabled": true},{"category": "ResourceHealth","enabled": true}]'
+ACTION=""
+DIAGNOSTIC_NAME=""
+STORAGE_ACCOUNT_ID=""
+
+while getopts "n:s:t:" flag; do
+    case "${flag}" in
+        s) SUBS=$(cat ${OPTARG} | sed 's/,/;/g');;
```

Comment:
The reason is to be able ot iterate each subscription id in the subscriptions list. The subscriptions list original format uses ',' and it's the working format for the ARM templates. 
Now here the separator defined on top is ';' because using ',' would conflict with LOGS and ROLES arguments. Azure expects LOGS in that format and cannot be changed, as for ROLES I'm using a mix of commas and semi-colon to diferentiate each role and their attributes.
This separator is used in the for loop to grab each id from the subs list and in the other script in the nested for loop to iterate each sub id and role.

Reply:
this way the script accepts the same subscriptions list as the ARM template

Reply:
I say conflict before because if the separator is ',' or ' ' it would trim the argument mid element for LOGS and ROLES